api/src/Repository/SortableServiceEntityRepository.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Repository;
+
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepositoryInterface;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use Gedmo\Sortable\Entity\Repository\SortableRepository;
+use LogicException;
+use function sprintf;
+
+/**
+ * Optional EntityRepository base class with a simplified constructor (for autowiring).
+ *
+ * To use in your class, inject the "registry" service and call
+ * the parent constructor. For example:
+ *
+ * class YourEntityRepository extends ServiceEntityRepository
+ * {
+ *     public function __construct(ManagerRegistry $registry)
+ *     {
+ *         parent::__construct($registry, YourEntity::class);
+ *     }
+ * }
+ *
+ * @template T
+ * @template-extends EntityRepository<T>
+ */
+abstract class SortableServiceEntityRepository extends SortableRepository implements ServiceEntityRepositoryInterface {
```

Comment:
Ich verstehe nicht ganz, was dieser Konstruktor mit "Sortable" zu tun hat. Bist du sicher die Klasse ist korrekt benannt?

Reply:
Ich füge folgenden Kommentar noch an den Anfang dazu. Macht's das klarer?

`This class is a 1:1 copy of Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository with the only change that it extends from Gedmo\Sortable\Entity\Repository\SortableRepository instead of the default Doctrine\ORM\EntityRepository in order to make this work with Sortable Doctrine extension.`